spacy/tests/pipeline/test_spancat.py

```diff
@@ -41,38 +41,42 @@ def make_examples(nlp, data=TRAIN_DATA):
     return train_examples
 
 
-def test_no_label():
+@pytest.mark.parametrize("name", ["spancat", "spancat_exclusive"])
```

Comment:
I don't care a whole lot about this, but it might make sense to have `["spancat", "spancat_exclusive"]` as a global variable instead of repeating it?